docs/development/writing_zeppelin_interpreter.md

```diff
@@ -118,6 +118,12 @@ Here is an example of `interpreter-setting.json` on your own interpreter.
       "language": "your-syntax-highlight-language",
       "editOnDblClick": false,
       "completionKey": "TAB"
+    },
+    "config": {
+      "fontSize": 9,
+      "colWidth": 12,
```

Comment:
Do we need to expose `fontSize` and `colWidth` to user to customize ? I don't see you need it here. If not, we could remove them, and expose them later in future if it is really necessary.  Because once we expose something to users, we have to maintain it and keep compatibility.

Reply:
Ok, I will remove it first, If need it later, can add it again.